demo/src/sections/Editor/Settings/Settings.js

```diff
@@ -41,19 +41,27 @@ const Settings = _ => {
     }
   }
 
-  function handleEditorDidMount(valueGetter, editor) {
-    setGetEditorValue(_ => valueGetter);
+  function getEditorValue() {
+    return editorRef.current?.getValue();
+  }
+
+  function handleEditorDidMount(editor, monaco) {
     setIsEditorReady(true);
     editorRef.current = editor;
   }
 
   function handleApply() {
+    const currentValue = getEditorValue();
     let oprions;
     try {
-      oprions = JSON.parse(getEditorValue());
+      oprions = JSON.parse(currentValue);
+      console.log(oprions);
```

Comment:
please remove the `console.log`

Reply:
I have removed the unnecessary `console.log`. Thank you for point it out.